sros/docker/launch.py

```diff
@@ -550,16 +554,18 @@ def bootstrap_config(self):
             for l in iter(gen_bof_config()):
                 self.wait_write(l)
 
+            # save bof config on disk
+            self.wait_write("/bof save")
+
             self.wait_write("/admin save")
             self.wait_write(
                 "/configure system management-interface configuration-mode {mode}".format(
                     mode=self.mode
                 )
             )
 
-        # save bof config on disk
-        self.wait_write("/bof save")
-        self.wait_write("/logout")
+            # logout at the end of execution 
+            self.wait_write("/logout")
```

Comment:
I move the /logout command to at the end of boostraping config exexuction, to ensure all comand in boostraping config are executed before logout from terminal.

Reply:
thanks, good catch.